src/sinks/influxdb/mod.rs

```diff
@@ -244,12 +244,16 @@ pub(in crate::sinks) fn influx_line_protocol(
     }
 
     encode_string(measurement, line_protocol);
-    line_protocol.put_u8(b',');
 
     // Tags
     let unwrapped_tags = tags.unwrap_or_default();
-    encode_tags(unwrapped_tags, line_protocol);
-    line_protocol.put_u8(b' ');
+    if unwrapped_tags.is_empty() {
+        line_protocol.put_u8(b' ');
+    } else {
+        line_protocol.put_u8(b',');
+        encode_tags(unwrapped_tags, line_protocol);
+        line_protocol.put_u8(b' ');
+    }
```

Comment:
nit: Could technically de-duplicate this trailing put:
```suggestion
    if !unwrapped_tags.is_empty() {
        line_protocol.put_u8(b',');
        encode_tags(unwrapped_tags, line_protocol);
    }
    line_protocol.put_u8(b' ');
```